CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 * [ENHANCEMENT] metrics-generator: expose span size as a metric [#1662](https://github.com/grafana/tempo/pull/1662) (@ie-pham)
 * [ENHANCEMENT] Set Max Idle connections to 100 for Azure, should reduce DNS errors in Azure [#1632](https://github.com/grafana/tempo/pull/1632) (@electron0zero)
+* [CHANGE] Use Parquet for local block search, tag search and tag value search instead of flatbuffers.
+  - Makes Parquet the default encoding.
+  - *BREAKING CHANGE* Local search for traces, tags and tag values no longer works with v2 blocks. It is recommended to manually 
+  clean out the search folder to remove no longer used files. By default these files are located at `/var/tempo/wal/search`.
```

Comment:
Is this manual delete actually needed?  I think the search files will be deleted alongside their parent block, should be ok, unless I am missing something.

Reply:
I think so. I'm pretty sure I removed the code that tracks and removes these files completely.

Reply:
Definitely misunderstood the way this works. No the delete was not necessary. Updated changelog.